x-pack/legacy/plugins/uptime/public/state/reducers/dynamic_settings.ts

```diff
@@ -31,23 +33,22 @@ export const dynamicSettingsReducer = handleActions<DynamicSettingsState, any>(
       ...state,
       loading: true,
     }),
-    [String(getDynamicSettingsSuccess)]: (state, action: Action<DynamicSettings>) => {
-      return {
-        loading: false,
-        settings: action.payload,
-      };
-    },
-    [String(getDynamicSettingsFail)]: (state, action: Action<Error>) => {
-      return {
-        loading: false,
-        loadError: action.payload,
-      };
-    },
+    [String(getDynamicSettingsSuccess)]: (state, action: Action<DynamicSettings>) => ({
+      ...state,
+      loading: false,
+      settings: action.payload,
+    }),
+    [String(getDynamicSettingsFail)]: (state, action: Action<Error>) => ({
+      ...state,
+      loading: false,
+      loadError: action.payload,
+    }),
     [String(setDynamicSettings)]: state => ({
       ...state,
       loading: true,
     }),
     [String(setDynamicSettingsSuccess)]: (state, action: Action<DynamicSettings>) => ({
+      ...state,
```

Comment:
it's not necessary to destructure prev state to return new state, if you know what new state can be in absolute terms.

Reply:
++ I removed these additions in https://github.com/elastic/kibana/pull/64111/commits/84129513f7463ec7dddf2b78be245291f3bb8517

I think initially I thought some state-related issue was happening because of the lack of `spread` on unreferenced state params, and then never removed them after the fact.